src/screens/SendToken/SendTokenAmount.js

```diff
@@ -175,23 +181,24 @@ class SendTokenAmount extends React.Component<Props, State> {
     });
   };
 
-  handleChange = (value: Object) => {
-    this.setState({ value });
+  handleChange = async (value: Object) => {
+    const gasLimit = await this.getGasLimit();
```

Comment:
Consider this approach:
```
handleChange = (value: Object) => {
  this.setState({ value });
  this.getGasLimit().then(gasLimit => this.setState({ gasLimit }));
};
```

Reply:
You need to call `this.setState({ value });` immediately in order to prevent having a lag once you type

Reply:
Fixed in newer commits 👍